src/elements/common/item/ItemSubDetails.js

```diff
@@ -33,15 +35,17 @@ const ItemSubDetails = ({ view, item }: Props) => {
 
     return (
         <span>
-            <FormattedMessage
-                {...message}
-                values={{
-                    date: DateValue,
-                    name: modifiedBy,
-                }}
-            />
-            <span>
-                &nbsp;-&nbsp;
+            <span className={isGridView ? 'bdl-ItemSubDetails-span-modifiedBy' : ''}>
```

Comment:
Class names should avoid mention of visual treatments or element names (span). Also, can we apply the class always and then override it based on the presence (or absence) class from a higher level, like `bdl-GridView`?